book/src/tutorial/generate_docs.md

```diff
@@ -0,0 +1,36 @@
+# Generating documentation
+And finally the last feature! Just run the following command in the folder of the safe wrapper crate:
+
+```sh
+gir -c Gir.toml -d ../gir-files --doc-target-path docs.md -m doc
+```
+
+* `-d ../gir-files`: flag to select the folder containing the .gir files.
+* `--doc-target-path docs.md`: flag to select the name of the markdown file containing the documentation.
+* `-m doc`: flag to select the work mode for generating the documentation.
+
+It'll generate a markdown file if everything went fine. It contains all of the crate's documentation. If you want to put it into your crate's source code like "normal" doc comments, run:
+
+```sh
+cargo install rustdoc-stripper
+rustdoc-stripper -g -o docs.md
+```
+
+And now your crate should be completely documented as expected!
+
+Running the above commands again would duplicate the doc comments. Make sure to first remove the old ones before running the command again. You can do this by running the following commands:
+
+```sh
+rustdoc-stripper -s -n
+rustdoc-stripper -g -o docs.md
+```
+
+Try building the documentation and also try it with the various features you might have
+
+```sh
+cargo doc
+```
+
+Congratulations, we are done! You have successfully created the safe wrapper for a C library!
+
+You can easily publish your generated bindings and the wrapper to crates.io to allow others to use it. Publishing crates is easy but keep in mind that they need to be maintained as well. We set up the project folder in a way that easily allows sharing the code. All that is needed is to add some information to your Cargo.toml. Gir will not override them when you re-generate bindings. Easy, right? If this is your first time publishing a crate, you can find a detailed guide [here](https://doc.rust-lang.org/cargo/reference/publishing.html).
```

Comment:
Another thing we could mention here in the future, is the usage of https://gitlab.gnome.org/World/Rust/gir-rustdoc & potentially having some CI templates for docs for both Github Actions & Gitlab

Reply:
That's a good suggestion. I was not aware of it. I opened https://github.com/gtk-rs/gir/issues/1396 for this. The PR already is fairly large and a lot to read and this is something that could be added at a later stage too. If it is okay with you, I would prefer to not include your suggested additional information in this PR?

Reply:
sure